data/avro-schema/src/main/resources/avro/net/corda/data/flow/state/session/SessionState.avsc

```diff
@@ -34,6 +34,15 @@
       "type": "boolean",
       "doc": "True if the user has set requireClose to be true when calling initiate flow. False otherwise."
     },
+    {
+      "name": "sessionTimeout",
```

Comment:
I don't think you need to add and a new avro field. You can just set it in the sessionProperties which are already exchanged on SessionEvents and then stored in the SessionState

Reply:
I was following changes related to `requireClose` as example. Is there some reason that filed was created as new avro field instead of setting it in sessionProperties?